src/core/ext/transport/binder/utils/check_binder_api.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GRPC_CORE_EXT_TRANSPORT_BINDER_UTILS_CHECK_BINDER_API_H
+#define GRPC_CORE_EXT_TRANSPORT_BINDER_UTILS_CHECK_BINDER_API_H
+
+#include <grpc/impl/codegen/port_platform.h>
+
+// This header defines GRPC_SUPPORT_BINDER_TRANSPORT when the necessary
+// NDK binder API for binder transport is available.
+
+#ifdef GPR_ANDROID
```

Comment:
If you move this into port_platform.h then we can eliminate the awkwardness everywhere where we need to include port_platform.h to check one thing, then include this, then check another thing... I'd prefer if we just include port_platform.h and then check the thing that we need to.